requirements.dev.txt

```diff
@@ -1,5 +1,8 @@
-rich
-pynacl
-aiofiles
+rich~=13.7.0
+pynacl~=1.5.0
+aiofiles~=23.2.1
 
-python3_indy
\ No newline at end of file
+python3_indy~=1.16.0.post286
```

Comment:
Did a quick scan, this doesn't seem to be imported anywhere - is it a transitive dependency?

Reply:
@esune `import indy` is used in the integration tests

Reply:
Will be great if there was an alternative way to achieve that integration testing, because figuring out how to install the libindy dependency on the docker image probably took me a good 2 hours of trial and error :joy:

That's why Dockerfile.test now builds from ubuntu:20.04 and installs python separately, because that `apt-get install libindy` doesn't work on debian images, and python docker images all use debian

Reply:
Should migrate to `indy-vdr` at some point in the near furture.